Handle external youtube playback blocked

Trace:
```PROLOG
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Unexpected exception
        at lavalink.client.player.event.AudioEventAdapterWrapped.onEvent(AudioEventAdapterWrapped.java:55)
        at lavalink.client.player.LavalinkPlayer.lambda$emitEvent$0(LavalinkPlayer.java:208)
        at java.base/java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:804)
        at lavalink.client.player.LavalinkPlayer.emitEvent(LavalinkPlayer.java:208)
        at lavalink.client.io.LavalinkSocket.handleEvent(LavalinkSocket.java:150)
        at lavalink.client.io.LavalinkSocket.onMessage(LavalinkSocket.java:102)
        at lavalink.client.io.ReusableWebSocket$DisposableSocket.onMessage(ReusableWebSocket.java:134)
        at org.java_websocket.client.WebSocketClient.onWebsocketMessage(WebSocketClient.java:462)
        at org.java_websocket.drafts.Draft_6455.processFrame(Draft_6455.java:696)
        at org.java_websocket.WebSocketImpl.decodeFrames(WebSocketImpl.java:367)
        at org.java_websocket.WebSocketImpl.decode(WebSocketImpl.java:212)
        at org.java_websocket.client.WebSocketClient.run(WebSocketClient.java:389)
        at java.base/java.lang.Thread.run(Thread.java:844)
Caused by: lavalink.client.io.RemoteTrackException: <a href="http://www.youtube.com/watch?v=eLcYmljHqqY&feature=player_embedded" target="_blank">Watch this video on YouTube.</a> Playback on other websites has been disabled by the video owner.
        at lavalink.client.io.LavalinkSocket.handleEvent(LavalinkSocket.java:136)
        ... 8 more
lavalink.client.io.RemoteTrackException: <a href="http://www.youtube.com/watch?v=eLcYmljHqqY&feature=player_embedded" target="_blank">Watch this video on YouTube.</a> Playback on other websites has been disabled by the video owner.
        at lavalink.client.io.LavalinkSocket.handleEvent(LavalinkSocket.java:136)
        at lavalink.client.io.LavalinkSocket.onMessage(LavalinkSocket.java:102)
        at lavalink.client.io.ReusableWebSocket$DisposableSocket.onMessage(ReusableWebSocket.java:134)
        at org.java_websocket.client.WebSocketClient.onWebsocketMessage(WebSocketClient.java:462)
        at org.java_websocket.drafts.Draft_6455.processFrame(Draft_6455.java:696)
        at org.java_websocket.WebSocketImpl.decodeFrames(WebSocketImpl.java:367)
        at org.java_websocket.WebSocketImpl.decode(WebSocketImpl.java:212)
        at org.java_websocket.client.WebSocketClient.run(WebSocketClient.java:389)
        at java.base/java.lang.Thread.run(Thread.java:844)
```

--- a/src/main/java/ml/duncte123/skybot/utils/AudioUtils.java
+++ b/src/main/java/ml/duncte123/skybot/utils/AudioUtils.java
@@ -240,9 +240,17 @@ public class AudioUtils {
 
             @Override
             public void loadFailed(FriendlyException exception) {
-                if (announce)
+                if (!announce) {
+                    return;
+                }
+                
+                if (exception.getMessage().endsWith("Playback on other websites has been disabled by the video owner.")) {
+                    sendEmbed(channel, embedField(embedTitle, "Could not play: " + trackUrl
+                        + "\nExternal playback of this video was blocked by YouTube."));
+                } else {
                     sendEmbed(channel, embedField(embedTitle, "Could not play: " + exception.getMessage()
                         + "\nIf this happens often try another link or join our [support guild](https://discord.gg/NKM9Xtk) for more!"));
+                }
             }
         });
     }

